blob-storage/Bean.go

```diff
@@ -66,8 +153,11 @@ type GcpBlobBaseConfig struct {
 type BlobStorageType string
 
 const (
-	BLOB_STORAGE_AZURE BlobStorageType = "AZURE"
-	BLOB_STORAGE_S3                    = "S3"
-	BLOB_STORAGE_GCP                   = "GCP"
-	BLOB_STORAGE_MINIO                 = "MINIO"
+	BLOB_STORAGE_AZURE            BlobStorageType = "AZURE"
+	BLOB_STORAGE_S3                               = "S3"
+	BLOB_STORAGE_GCP                              = "GCP"
+	BLOB_STORAGE_MINIO                            = "MINIO"
+	BlobStorageObjectTypeCache                    = "cache"
+	BlobStorageObjectTypeArtifact                 = "artifact"
+	BlobStorageObjectTypeLog                      = "log"
```

Comment:
these 3 are not blobStorageType

Reply:
that's why I've used BlobStorageObjectTypeCache   "object" here not simply  blobStorageType, if it makes any other sense let's discuss over the naming